test/hotspot/jtreg/compiler/compilercontrol/commands/MemStatTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Thanks! But I think we're OK with Redhat taking credit for this one :).

Reply:
Oops :-)